manifests/certonly.pp

```diff
@@ -131,7 +131,7 @@
   Optional[String[1]]                       $cron_success_command = undef,
   Array[Variant[Integer[0, 59], String[1]]] $cron_monthday        = ['*'],
   Variant[Integer[0,23], String, Array]     $cron_hour            = fqdn_rand(24, $title),
-  Variant[Integer[0,59], String, Array]     $cron_minute          = fqdn_rand(60, fqdn_rand_string(10, $title)),
+  Variant[Integer[0,59], String, Array]     $cron_minute          = fqdn_rand(60, fqdn_rand_string(10, undef, $title)),
```

Comment:
Is the usage of `fqdn_rand_string()` needed?  I feel like we just want the `$cron_minute` to be different for each resource so we can do:

```suggestion
  Variant[Integer[0,59], String, Array]     $cron_minute          = fqdn_rand(60, $title),
```

See https://puppet.com/docs/puppet/7/function.html#fqdn-rand

Reply:
Hmm, good question.  The `fqdn_rand_string()` call was added in https://github.com/voxpupuli/puppet-letsencrypt/commit/aeed58ae, whose commit message says:

> fixed randomness if the domain is almost same, like example.com and www.example.com/

I performed some tests:

```console
$ puppet apply -e 'notice(fqdn_rand(24, "example.com")) notice(fqdn_rand(60, "example.com"))'
Notice: Scope(Class[main]): 21
Notice: Scope(Class[main]): 39

$ puppet apply -e 'notice(fqdn_rand(24, "www.example.com")) notice(fqdn_rand(60, "www.example.com"))'
Notice: Scope(Class[main]): 8
Notice: Scope(Class[main]): 30

$ puppet apply -e 'notice(fqdn_rand(24, "www.example.com/")) notice(fqdn_rand(60, "www.example.com/"))'
Notice: Scope(Class[main]): 10
Notice: Scope(Class[main]): 52
```

Each of the hour/minute values is unique.  At least with the current implementation of `fqdn_rand()`, it looks like that's no longer a concern, so I'll go ahead and remove the `fqdn_rand_string()`.

Reply:
28d67f5